tozny/resource_realm_group.go

```diff
@@ -137,6 +287,23 @@ func resourceRealmGroupRead(ctx context.Context, d *schema.ResourceData, m inter
 
 	d.Set("name", group.Name)
 
+	// Only reading access policies attached to this group
+	groupIDs := []string{d.Get("group_id").(string)}
+	listAccessPoliciesRequest := identityClient.ListAccessPoliciesRequest{
+		RealmName: d.Get("realm_name").(string),
+		GroupIDs:  groupIDs,
+	}
+
+	realmAccessPolicies, err := toznySDK.ListAccessPolicies(ctx, listAccessPoliciesRequest)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	accessPolicies := flattenAccessPolicyItems(realmAccessPolicies.GroupAccessPolicies[0]) // Only one Group's access policies were requested
```

Comment:
your comment makes me feel like that is wrong, but isnt it correct?

Reply:
Comment was meant to explain the reasoning behind only taking the first element from the group access policies list, `GroupAccessPolicies[0]`. I can make it make clarifying

Reply:
The response from `ListAccessPolicies` (which takes a list of group ids) is a list `realmAccessPolicies`. We only requested a single group id, so the list is only of length 1. Wanted to avoid confusion in the future when somebody comes across this and wonders "why only the first element"